crates/graphql/src/schema/objects/auction_house.rs

```diff
@@ -135,4 +139,27 @@ impl AuctionHouse {
             .await
             .map_err(Into::into)
     }
+
+    pub async fn volume(
+        &self,
+        ctx: &AppContext,
+        start_date: Option<DateTime<Utc>>,
+        end_date: Option<DateTime<Utc>>,
+    ) -> FieldResult<Option<U64>> {
+        let conn = ctx.shared.db.get()?;
+
+        let result = queries::auction_house::volume(
+            &conn,
+            &self.address,
+            start_date.map(|v| v.naive_utc()),
+            end_date.map(|v| v.naive_utc()),
+        )?;
+
+        Ok(result
+            .into_iter()
+            .next()
+            .map(|models::AuctionHouseVolume { volume }| {
+                volume.to_u64().unwrap_or_default().into()
```

Comment:
@ray-kast is this correct?

Reply:
@ray-kast Using `volume.to_u64().ok_or_else(|| anyhow!("..."))` throws mismatched types error:
```
mismatched types
expected enum `std::result::Result<std::option::Option<bigint::U64>, juniper::FieldError>`
   found enum `std::result::Result<std::option::Option<u64>, anyhow::Error>`
```

Reply:
@ray-kast That worked. Thanks!